hooks/install-hooks.sh

```diff
@@ -3,14 +3,19 @@
 HOOK_NAMES="pre-commit"
 HOOK_DIR=$(git rev-parse --show-toplevel)/.git/hooks
 INSTALL_DIR=$(git rev-parse --show-toplevel)/hooks
-
+COLOR_GREEN=`tput setaf 2`
+COLOR_RESET=`tput sgr0`
```

Comment:
Can you add an empty line between the variables and the loop? Easier to read imo.